pom.xml

```diff
@@ -31,7 +31,7 @@
 		<main.basedir>${basedir}</main.basedir>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<java.version>1.8</java.version>
-		<eclipse.repository>https://download.eclipse.org/releases/2019-03/201903201000</eclipse.repository>
+		<eclipse.repository>http://download.eclipse.org/releases/2019-09/</eclipse.repository>
```

Comment:
This will introduce the risk of a MITM attack. We need an HTTPs URL for the repository.

Reply:
Yes, definitely. My bad. To be corrected...